gettsim/benefits/kinderzuschlag.py

```diff
@@ -192,79 +193,8 @@ def _calc_kiz_regel_since_2011(household, params):
     ]
 
 
-def get_wohnbedarf(yr):
-    """ Specifies the percent share of living costs that is attributed to the parents
-        This is a share that is defined by the "Existenzminimumsbericht".
-
-        The actual tables are found in the official "Merkblatt Kinderzuschlag".
-    """
-    # cols: number of adults
-    # rows: number of kids
-    wohnbedarf = {
-        "2011": [
-            [75.90, 83.11],
-            [61.16, 71.10],
-            [51.21, 62.12],
-            [44.05, 55.15],
-            [38.65, 49.59],
-        ],
-        "2012": [
-            [76.34, 83.14],
-            [61.74, 71.15],
-            [51.82, 62.18],
-            [44.65, 55.22],
-            [39.23, 49.66],
-        ],
-        "2013": [
-            [76.34, 83.14],
-            [61.74, 71.15],
-            [51.82, 62.18],
-            [44.65, 55.22],
-            [39.23, 49.66],
-        ],
-        "2014": [
-            [76.69, 83.30],
-            [62.20, 71.38],
-            [52.31, 62.45],
-            [45.13, 55.50],
-            [39.69, 49.95],
-        ],
-        "2015": [
-            [76.69, 83.30],
-            [62.20, 71.38],
-            [52.31, 62.45],
-            [45.13, 55.50],
-            [39.69, 49.95],
-        ],
-        "2016": [
-            [77.25, 83.16],
-            [62.93, 71.17],
-            [53.09, 62.20],
-            [45.92, 55.24],
-            [40.45, 49.69],
-        ],
-        "2017": [
-            [77.25, 83.16],
-            [62.93, 71.17],
-            [53.09, 62.20],
-            [45.92, 55.24],
-            [40.45, 49.69],
-        ],
-        "2018": [
-            [77.24, 83.25],
-            [62.92, 71.30],
-            [53.08, 62.36],
-            [45.90, 55.41],
-            [40.43, 49.85],
-        ],
-        "2019": [
-            [77.10, 83.60],
-            [62.73, 71.93],
-            [52.88, 62.96],
-            [45.70, 56.04],
-            [40.24, 50.49],
-        ],
-        "2020": [[77, 84], [63, 72], [53, 63], [46, 56], [40, 50]],
-    }
-
-    return wohnbedarf[str(yr)]
+def kiz_dummy(household, params, arbeitsl_geld_2_params):
```

Comment:
Maybe we could raise a simple warning saying that there was no kinderzuschlag and we didn't calculate anything instead!